search.html

```diff
@@ -0,0 +1,263 @@
+	</script>
+	<script id="js2" type="text/javascript" language="JavaScript" src="doc/js/topbar.js">
+	</script>
+	<script src="https://cdn.jsdelivr.net/npm/algoliasearch@4.5.1/dist/algoliasearch-lite.umd.js" integrity="sha256-EXPXz4W6pQgfYY3yTpnDa3OH8/EPn16ciVsPQ/ypsjk=" crossorigin="anonymous"></script>
+	<script src="https://cdn.jsdelivr.net/npm/instantsearch.js@4.8.3/dist/instantsearch.production.min.js" integrity="sha256-LAGhRRdtVoD6RLo2qDQsU2mp+XVSciKRC8XPOBWmofM=" crossorigin="anonymous"></script>
+	<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/instantsearch.css@7.4.5/themes/satellite-min.css" integrity="sha256-TehzF/2QvNKhGQrrNpoOb2Ck4iGZ1J/DI4pkd2oUsBc=" crossorigin="anonymous"> </head>
+
+<style>
+.main-description { 
+  margin: 1em 0
+}
+
+.DocSearch-Button { 
+    margin: 0 0 0 0px !important; 
+}
+
+.search-box { 
+    margin-left: 0.5rem;
+}
+
+.search-hits { 
+    margin-left: 0.5rem;
+}
+
+.search-pagination { 
+    margin-top:0.3rem;
+    margin-left:0.5rem;
+    margin-bottom:1rem;
+}
+</style>
+
+<body>
+	<div id="container">
+		<div class="topbar">
+			<div class="topbar-expand">
+				<button onclick="toggleDisplay();">
+					<svg version="1.1" id="Capa_1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 54 54" width="24" height="24">
+						<g>
+							<path style="fill:#000000;" d="M27,54c-0.552,0-1-0.448-1-1V8c0-0.552,0.448-1,1-1s1,0.448,1,1v45C28,53.552,27.552,54,27,54z" />
+							<path style="fill:#000000;" d="M11,25c-0.256,0-0.512-0.098-0.707-0.293c-0.391-0.391-0.391-1.023,0-1.414l16-16
+                                                     c0.391-0.391,1.023-0.391,1.414,0s0.391,1.023,0,1.414l-16,16C11.512,24.902,11.256,25,11,25z" />
+							<path style="fill:#000000;" d="M43,25c-0.256,0-0.512-0.098-0.707-0.293l-16-16c-0.391-0.391-0.391-1.023,0-1.414
+                                                     s1.023-0.391,1.414,0l16,16c0.391,0.391,0.391,1.023,0,1.414C43.512,24.902,43.256,25,43,25z" />
+							<path style="fill:#000000;" d="M43,2H11c-0.552,0-1-0.448-1-1s0.448-1,1-1h32c0.552,0,1,0.448,1,1S43.552,2,43,2z" /> </g>
+					</svg>
+				</button>
+			</div>
+			<div class="topbar-title">
+				<h1>
+                      Erlang/OTP 24.1.4
+                  </h1> </div>
+			<div class="search-expand">
+				<button id="docsearch-mobile">
+					<svg fill="#000000" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 50 50" width="24" height="24">
+						<path d="M 21 3 C 11.621094 3 4 10.621094 4 20 C 4 29.378906 11.621094 37 21 37 C 24.710938 37 28.140625 35.804688 30.9375 33.78125 L 44.09375 46.90625 L 46.90625 44.09375 L 33.90625 31.0625 C 36.460938 28.085938 38 24.222656 38 20 C 38 10.621094 30.378906 3 21 3 Z M 21 5 C 29.296875 5 36 11.703125 36 20 C 36 28.296875 29.296875 35 21 35 C 12.703125 35 6 28.296875 6 20 C 6 11.703125 12.703125 5 21 5 Z" />
+					</svg>
+				</button>
+			</div>
+		</div>
+		<aside id="leftnav">
```

Comment:
A good TODO would be to move all this into a template, as right now, if we need to make a change to the side bar, it'll have to be done in at least two places, but also probably something worth doing in another PR.